src/tooltip.tsx

```diff
@@ -619,15 +618,15 @@ const Tooltip = ({
     return (
         <BaseTooltip
             content={
-                <Box className={styles.content}>
```

Comment:
This was working just by luck. The Box component sets padding to 0 by default, and in this case the padding is being applied also through className prop.

When changing the logic in the Box CSS file to use vars instead of sprinkles, the padding from props took precedence over the one passed through className (I guess it depends on how vanilla determines the one that "wins").

In this case, given that we are not using padding props at all, it doesn't actually makes sense to use Box here. I've replaced it with a div to avoid the issue I've mentioned before

Reply:
I've also detected some use cases in webapp where padding is being applied through className instead of props. These will fail because of the same reason, so we may need to fix them after merging this PR:

- web/src/pages/cancellation-with-retention-v3/v1/components/list-offers/index.tsx
- web/src/pages/mv-forms/forms/cancelamento-sva/commons/infos/index.tsx
- web/src/pages/mv-forms/forms/problemas-cobrancas/commons/infos/index.tsx
- web/src/pages/mv-forms/forms/suporte-tecnico/commons/infos/index.tsx
- web/src/pages/mv-forms/forms/suspensao-temporaria/commons/choice/index.tsx
- web/src/pages/mv-forms/forms/suspensao-temporaria/commons/infos/index.tsx
- web/src/pages/update-contacts-vivo-code/components/update-contact-parcial/index.tsx

Reply:
As commented in teams, I'd deprecate the className prop in the Box component. If someone wants a div with styles, then they can use a div.

Reply:
Done, deprecated className